### DISCUSSION TITLE
Undiscord tries to delete messages from a bot over and over.

### DISCUSSION THREAD

**Discussion by tanyamga** [victornpb/undiscord#501] [1. Questions & Answers]
Im trying to delete all my messages in a server, the server has a bot that would "snipe" your deleted messages when a command was entered and people did that a lot to me, now when i try to delete MY messages, it also tries to delete the snipebot's messages since theyre attached to me somehow, how do i fix this to where the bot only tries to delete messages that come from me?

" [6/1909] 5/1/2023, 10:23:26 PM SnipeBot#3414: 
Error deleting message, API responded with status 403!	{"message":"Missing Permissions","code":50013} "

**Comment by iSeeeker** [comment]
Not sure what the fix is to this, but it's ruining everything

**Comment by ikno96** [comment]
Is there any solution to this problem? @victornpb

**Comment by wtweeku** [comment]
Is there any solution to this problem? @victornpb

**Comment by 573rn** [comment]
If the 403 error is because of a bot:

In Regex enter this:

`^(?!\s*$)(?!.*\bwarn\b).*$`

This works specifically with a GPT bot on the server I'm in.

(?!\s*$) = Whitespace: the bot in my server often posts posts that are only whitespace, aka: no characters at all. This skips posts with no input.

(?!.*\bwarn\b) = The other post the bot makes contains the word "Warn", this skips all posts that contain the word "Warn"

Replace these with whatever unique words or phrases the bot in the server you are trying to avoid uses.

**Reply by ghost** [reply]
Hey, where do I input this?
